README.md

```diff
@@ -32,6 +37,8 @@ It supports the following outputs:
   * HTSP (own protocol)
   * SAT>IP
 
+Also can work with XMLTV EPG format
```

Comment:
"Also it can work with XMLTV EPG format" would probably be better here

Reply:
Acceptable, also with xmltv.sock